google/cloud/spanner_v1/__init__.py

```diff
@@ -79,6 +80,7 @@
 This value can only be used for timestamp columns that have set the option
 ``(allow_commit_timestamp=true)`` in the schema.
 """
+TransactionTypes = DirectedReadOptions.ReplicaSelection.Type
```

Comment:
Do we need to create this type ? Can't customers directly import `DirectedReadOptions.ReplicaSelection.Type` 

We are not using this anywhere in the source code

Reply:
I think we don't need an additional type for this.
Instead customers can do the following,
```
from google.cloud.spanner_v1 import DirectedReadOptions
directed_read_options_for_request = {
    "include_replicas": {
        "replica_selections": [
            {
                "type_": 
    directed_read_options_for_request = {
        "include_replicas": {
            "replica_selections": [
                {
                    "type_": DirectedReadOptions.ReplicaSelection.Type.READ_ONLY,
                },
            ],
            "auto_failover_disabled": True,
        },
    },
            },
        ],
        "auto_failover_disabled": True,
    },
}
```

Does this look correct?

Reply:
yes